app/controllers/static_pages_controller.rb

```diff
@@ -1,10 +1,10 @@
 class StaticPagesController < ApplicationController
-    def home
-    end
-    
-    def help
-    end
+  def home
```

Comment:
```suggestion
  def home; end
```
tương tự các method khác